Dynavity/Dynavity/persistence/data-mapping/Mappable.swift

```diff
@@ -0,0 +1,5 @@
+protocol Mappable {
```

Comment:
From what I can tell each DTO struct has a `Codable` extension. Does this requirement need to be captured in `Mappable`?

Reply:
I don't think it should. `Mappable`s  should not necessarily be `Codable`s. It depends on how we want to store the underlying `Mappable`s. 
E.g. If we were to use a database, then it wouldn't be a `Codable`, but probably something else that can be converted into a database row